solana/modules/nft_bridge/program/src/api/transfer.rs

```diff
@@ -137,7 +136,7 @@ pub fn transfer_native(
         return Err(TokenNotNFT.into());
```

Comment:
The renaming of spl_token_metadata to mpl_token_metadata may have added some confusion here. The derivation is the same because it is the same program ID.